src/skeptic/lib.rs

```diff
@@ -765,14 +769,15 @@ fn test_omitted_lines() {
 #[test]
 fn test_markdown_files_of_directory() {
     let files = vec![
-        "../../CHANGELOG.md",
-        "../../README.md",
-        "../../README.md.skt.md",
-        "../../template-example.md",
-        "../../tests/hashtag-test.md",
-        "../../tests/section-names.md",
-        "../../tests/should-panic-test.md",
+        vec!["CHANGELOG.md"],
+        vec!["README.md"],
+        vec!["README.md.skt.md"],
+        vec!["template-example.md"],
+        vec!["tests", "hashtag-test.md"],
+        vec!["tests", "section-names.md"],
+        vec!["tests", "should-panic-test.md"],
     ];
+    let files: Vec<PathBuf> = files.iter().map(PathBuf::from).collect();
```

Comment:
no need to explicitly state the `Vec<PathBuf>`. you can use `Vec<_>` and compiler will infer the types for you.